.scala-steward.conf

```diff
@@ -0,0 +1 @@
+updates.ignore = [ { groupId = "edu.berkeley.cs", artifactId = "firrtl" } ]
```

Comment:
can you confirm that artifactId is firrtl and not firrtl-interpreter. Not sure what it means

Reply:
It's the artifact id of a dependency. Click on `Ignore future updates` in #170